src/neuroconv/datainterfaces/behavior/audio/audiointerface.py

```diff
@@ -0,0 +1,162 @@
+from pathlib import Path
+from typing import Optional
+from warnings import warn
+
+from scipy.io.wavfile import read
+
+from neuroconv.basedatainterface import BaseDataInterface
+from neuroconv.datainterfaces.behavior.video.videodatainterface import _check_duplicates
+from neuroconv.tools.audio import add_acoustic_waveform_series
+from neuroconv.tools.nwb_helpers import (
+    make_or_load_nwbfile,
+)
+from neuroconv.utils import (
+    get_schema_from_hdmf_class,
+    get_base_schema,
+    OptionalFilePathType,
+)
+from pynwb import NWBFile, TimeSeries
+
+
+class AudioInterface(BaseDataInterface):
```

Comment:
interface can be removed

Reply:
Ah okay, I thought we only do that for Extractors. Thank you!